examples/python/human_pose_tracking/main.py

```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
     return str(destination_path)
 
 
+def get_blueprint() -> rrb.BlueprintLike:
+    return rrb.Viewport(
```

Comment:
`rrb.Viewport` is redundant here, you can just pass a `Horizontal` out

Reply:
👍🏻